mix.exs

```diff
@@ -50,14 +51,16 @@ defmodule Spacesuit.Mixfile do
     [
       # All envs
       {:hackney, "~> 1.7.1", override: true},
-      {:cowboy, github: "extend/cowboy"},
+      {:cowboy, "~> 2.5", override: true},
       {:poison, "~> 3.0", override: true},
       {:jsx, "~> 2.8.0"},
       {:joken, "~> 1.4.1"},
+      {:distillery, "~> 1.5.5"},
```

Comment:
Spacesuit wasn't written to use releases and won't work that way without a fair bit of work.